model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -45,20 +46,21 @@ def pattern(self):
         return dict(
             nodes=[
                 ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3'],
-                 is_training=True))],
+                            is_training=True))],
```

Comment:
Your changes will be in a conflict with @evolosen changes. Make sure that you are aligned with him.